observability/sops/alerts/strimzi_operator_container_frequently_restarting.adoc

```diff
@@ -0,0 +1,146 @@
+strimzi-cluster-operator-6ddcb45f47-2jpp2 strimzi-cluster-operator 	at io.vertx.core.impl.EventLoopContext.emit(EventLoopContext.java:22) ~[io.vertx.vertx-core-4.2.4.jar:4.2.4]
+strimzi-cluster-operator-6ddcb45f47-2jpp2 strimzi-cluster-operator 	at io.vertx.core.impl.AbstractContext.emit(AbstractContext.java:53) ~[io.vertx.vertx-core-4.2.4.jar:4.2.4]
+strimzi-cluster-operator-6ddcb45f47-2jpp2 strimzi-cluster-operator 	at io.vertx.core.impl.EventLoopContext.emit(EventLoopContext.java:22) ~[io.vertx.vertx-core-4.2.4.jar:4.2.4]
+strimzi-cluster-operator-6ddcb45f47-2jpp2 strimzi-cluster-operator 	at io.vertx.core.impl.VertxImpl$InternalTimerHandler.run(VertxImpl.java:883) ~[io.vertx.vertx-core-4.2.4.jar:4.2.4]
+strimzi-cluster-operator-6ddcb45f47-2jpp2 strimzi-cluster-operator 	at io.netty.util.concurrent.PromiseTask.runTask(PromiseTask.java:98) ~[io.netty.netty-common-4.1.71.Final.jar:4.1.71.Final]
+strimzi-cluster-operator-6ddcb45f47-2jpp2 strimzi-cluster-operator 	at io.netty.util.concurrent.ScheduledFutureTask.run(ScheduledFutureTask.java:176) ~[io.netty.netty-common-4.1.71.Final.jar:4.1.71.Final]
+strimzi-cluster-operator-6ddcb45f47-2jpp2 strimzi-cluster-operator 	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164) ~[io.netty.netty-common-4.1.71.Final.jar:4.1.71.Final]
+strimzi-cluster-operator-6ddcb45f47-2jpp2 strimzi-cluster-operator 	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469) ~[io.netty.netty-common-4.1.71.Final.jar:4.1.71.Final]
+strimzi-cluster-operator-6ddcb45f47-2jpp2 strimzi-cluster-operator 	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:503) ~[io.netty.netty-transport-4.1.71.Final.jar:4.1.71.Final]
+strimzi-cluster-operator-6ddcb45f47-2jpp2 strimzi-cluster-operator 	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986) ~[io.netty.netty-common-4.1.71.Final.jar:4.1.71.Final]
+strimzi-cluster-operator-6ddcb45f47-2jpp2 strimzi-cluster-operator 	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[io.netty.netty-common-4.1.71.Final.jar:4.1.71.Final]
+strimzi-cluster-operator-6ddcb45f47-2jpp2 strimzi-cluster-operator 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[io.netty.netty-common-4.1.71.Final.jar:4.1.71.Final]
+strimzi-cluster-operator-6ddcb45f47-2jpp2 strimzi-cluster-operator 	at java.lang.Thread.run(Thread.java:829) ~[?:?]
+I
+----
++
+In case yu see something like `Thread blocked` as above, then the reason is likely related to GC taking too much time to perform its taska hence, the issue may be related to memory limits being too low.
++
+This can be confirmed by looking at the various jvm entries in prometheus/grapahana, as example, you may get entries like the ones below:
```

Comment:
```suggestion
This can be confirmed by looking at the various JVM entries in Prometheus/Grafana. For example, you may get entries like the ones below:
```